app/workers/publishing_api_legacy_tags_worker.rb

```diff
@@ -0,0 +1,49 @@
+class PublishingApiLegacyTagsWorker
+  API = Services.publishing_api
+
+  def perform(edition_id, taxon_ids)
```

Comment:
I'd be more comfortable if this worker didn't require state on the queue, as this leaves the possibility open for jobs to execute out of insertion order, and for old data to be persisted. I think it might be better to lookup the taxons for the edition as part of the job, as this means that this is less of a problem?

Reply:
They are stored in the Publishing API, and I think also storing them temporarily in the Whitehall Sidekiq queue could lead to quickly changing the taxons for some content causing a race condition in updating the legacy tagging information.

Say the job fails, as the Publishing API errors. I think Sidekiq will retry this later, which makes this scenario even more likely.

Reply:
Ah yes, sorry. I see - will do.